docs/man/plastic.1

```diff
@@ -0,0 +1,88 @@
+PLASTIC(1)      General Commands Manual      PLASTIC(1)
+
+NAME
+
+    plastic - NES emulator with both graphical (GUI) and terminal (TUI) interfaces.
+
+SYNOPSIS
+
+    plastic 
```

Comment:
It actually can take an optional argument, which will make it start that game rightaway.

You can add here and also in the descriptions below

Reply:
Done